samples/cellular/nrf_cloud_multi_service/README.rst

```diff
@@ -520,9 +520,13 @@ Next, complete the following steps for each device you wish to provision:
    Do not attempt to use this example directly, it has been filled with dummy data.
    See `nRF Cloud REST API ProvisionDevices`_ for more details.
 
-#. Navigate to the `Provision Devices <nRF Cloud Portal Provision Devices_>`_ page of the nRF Cloud portal and upload the :file:`<device_uuid>_cred.csv` file you created.
+#. Navigate to the **Devices** page of the nRF Cloud portal and proceed as follows:
```

Comment:
The Getting started page uses "Click :guilabel:`Devices` under :ref:`Device Management` in the navigation pane on the left" if we want to align the wording getting to the Devices page.